src/parsers/sample-models.cpp

```diff
@@ -18,184 +18,153 @@
 
 #include "pinocchio/parsers/sample-models.hpp"
 
-#ifdef WITH_HPP_FCL
-#include <hpp/fcl/shape/geometric_shapes.h>
-#endif
-
 namespace se3
 {
   namespace buildModels
   {
+    
+    template<typename JointModel>
+    void addJointAndBody(Model & model, const JointModelBase<JointModel> & joint, const std::string & parent_name, const std::string & name)
```

Comment:
Keep it as it is, just add the keyword static to make sure that it is 
not usable outside the .cpp file.